pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools", "wheel"]
+requires = ["setuptools", "wheel", "cmake", "nanobind"]
```

Comment:
These should be added to a `pyproject.toml` specific for the wave package, not to the one of iree-turbine.